BUILD.bazel

```diff
@@ -195,6 +195,8 @@ load("@bazel_skylib//lib:dicts.bzl", "dicts")
 build_other_images_default = {
     # cmd images
     "$(container_prefix)/$(image_prefix)sidecar-shim:$(container_tag)": "//cmd/sidecars:sidecar-shim-image",
+    "$(container_prefix)/$(image_prefix)virt-exportproxy:$(container_tag)": "//cmd/virt-exportproxy:virt-exportproxy-image",
+    "$(container_prefix)/$(image_prefix)virt-exportserver:$(container_tag)": "//cmd/virt-exportserver:virt-exportserver-image",
```

Comment:
Not sure that I would classify these as "other" images as they are created/managed by virt-operator. Was thinking this is a better place: https://github.com/jschintag/kubevirt/blob/virt-exportproxy/hack/bazel-build-images.sh#L53-L54

Reply:
Yes, i was unsure where to add them. In the end i choose here, since that keeps the bash cli command looking cleaner.
But if you want them there @mhenriks i can move them to the script.

Reply:
Please move to `bazel-build-images.sh` where `virt-launcher` and such are build

Reply:
I have moved the 2 to the script and verified locally that they are built.
Please check if it is as expected @xpivarc .